aten/src/ATen/DeviceAccelerator.cpp

```diff
@@ -2,7 +2,7 @@
 #include <ATen/DeviceAccelerator.h>
 namespace at {
 
-C10_API std::optional<DeviceType> getAccelerator(bool checked) {
+std::optional<c10::DeviceType> getAccelerator(bool checked) {
```

Comment:
Why did you remove the C10_API here?

Reply:
Since there exists  [TORCH_API](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/DeviceAccelerator.h#L25) in header file. I just thought it was a mistake. Will add it back if it causes issues.